airflow/api_connexion/endpoints/request_dict.py

```diff
@@ -18,9 +18,14 @@
 
 from typing import Any, Mapping, cast
 
+from airflow.api_connexion.exceptions import BadRequest
+
 
 def get_json_request_dict() -> Mapping[str, Any]:
     """Cast request dictionary to JSON."""
     from flask import request
 
+    if not request.data:
+        raise BadRequest(detail="POST Body must not be None")
```

Comment:
```suggestion
        raise BadRequest(detail="POST Body must not be empty")
```

Probably more neutral? The “None” part may not make sense for clients that are not implemented in Python.

Reply:
Fixed all static checks & even changed the error message. Think we should be good to go, can you please trigger the static workflow one more time?

Reply:
Seems like the job is still failing, however I can't figure out where - do you see something wrong @uranusjr ?

Reply:
Any updates on this?